json-schemas/workflowRunsRequest.json

```diff
@@ -146,6 +146,9 @@
                     }
                 }
             }
+        },
+        "authenticityToken": {
+            "type": "string"
```

Comment:
The `valid_consensus_genome_workflow_runs` rails request requires this authenticityToken, is this a problem for the other uses of this endpoint?

Reply:
It failed on your local right? `/workflow_runs.json` has been working for me just fine. I forget, is `authenticityToken` only for mutations?

Reply:
ya, I think it is just for mutations